app/app.js

```diff
@@ -98,7 +98,9 @@ angular.module('app',
         CLAIMED_USER_SCOPES      : claimedAnonymousScopes.concat('dataproviders'),
         MAPBOX_API_KEY           : window.ushahidi.mapboxApiKey || 'pk.eyJ1IjoidXNoYWhpZGkiLCJhIjoiY2lxaXUzeHBvMDdndmZ0bmVmOWoyMzN6NiJ9.CX56ZmZJv0aUsxvH5huJBw', // Default OSS mapbox api key
         TOS_RELEASE_DATE         : new Date(window.ushahidi.tosReleaseDate).toJSON() ? new Date(window.ushahidi.tosReleaseDate) : false, // Date in UTC
-        PLATFORM_WEBSOCKET_REDIS_ADAPTER_URL : platform_websocket_redis_adapter_url
+        PLATFORM_WEBSOCKET_REDIS_ADAPTER_URL : platform_websocket_redis_adapter_url,
+        EXPORT_POLLING_INTERVAL  : window.ushahidi.export_polling_interval || 30000,
+        EXPORT_POLLING_COUNT     : window.ushahidi.export_polling_count || 50
```

Comment:
I added a limit on how many time the polling is done, not sure if that is needed but I don't want it to go on forever...